### PR TITLE
DDF-3884 Terminates DDF if any security policy file is malformed.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Prints the name of the failing policy file to the command line and executes a hard system exit.

This prevents the system from initializing in an unsecured state.

#### Who is reviewing it? 
@brjeter @blen-desta 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@rzwiefel
@stustison

#### How should this be tested? (List steps with links to updated documentation)
Edit any policy file in the `security` directory to make it badly formatted. Attempt to start the server. It should exit immediately, printing the name of the invalid file to the command line.

Will fail fast on the first badly formatted file it encounters.

#### What are the relevant tickets?
[DDF-3884](https://codice.atlassian.net/browse/DDF-3884)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java [hunk 1]
```diff
@@ -66,7 +67,15 @@ public void start(BundleContext bundleContext) throws Exception {
     File policyDirFile = new File(policyDir);
     List<ParsedPolicy> parsedPolicies = new ArrayList<>();
     for (File file : Objects.requireNonNull(policyDirFile.listFiles())) {
-      parsedPolicies.add(new Parser(false).parse(file));
+      ParsedPolicy parse = null;
+      try {
+        parse = new Parser(false).parse(file);
+      } catch (Exception e) {
+        System.out.printf("%nUnable to parse policy file, %s. Please fix and try again.%n", file);
+        System.exit(1);
```
Comment (brjeter) [inline platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java]: ❓ Should we not shutdown the system in a more official way like this? https://github.com/codice/ddf/blob/master/platform/security/secure-boot/src/main/java/org/codice/ddf/security/boot/SecureBoot.java#L117-L126

Comment (coyotesqrl) [reply platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java]: I can give it a shot, but my guess is that we're actually too early in the process and are best served with a hard exit.

Comment (rzwiefel) [reply platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java]: Exit code 1? I'd say we pick a unique exit code (I imagine 1 is already overloaded with purposes)

Comment (coyotesqrl) [reply platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java]: I wasn't particularly concerned with the exit code, other than it be non-zero. We certainly _can_ pick a specific value, but this is a fast failure condition and I'm not sure there's a lot of value in encoding the reason.

**Comment by coyotesqrl** [comment]
Updated with unit test.

**Comment by coyotesqrl** [comment]
build now

**Comment by brjeter** [comment]
Hero'd ✅

**Comment by coyotesqrl** [comment]
Accidental premature merge...mistakenly thought @rzwiefel had already approved (and failed to look to see his checkmark was missing).

Oops.

**Comment by rzwiefel** [comment]
Post merge review looks good tho ✅ 
Could possibly use System.err to print instead of System.out but not sure if it makes a big enough difference to matter.

**Pull request merged**